tests/common/test_utils.py

```diff
@@ -276,9 +281,17 @@ def construct_perf_analyzer_config(
 
     if request_rate:
         pa_config._args["request-rate-range"] = request_rate
+    elif llm_search_mode:
+        pa_config._args["periodic-concurrency-range"] = concurrency
     else:
         pa_config._args["concurrency-range"] = concurrency
 
+    if llm_search_mode:
+        pa_config._args["request-parameter"] = (
+            "max_token:" + str(max_token_count) + ":int"
+        )
+        pa_config._args["input-data"] = DEFAULT_INPUT_JSON_PATH + "/input-data.json"
```

Comment:
Quick heads up: This [line](https://github.com/triton-inference-server/model_analyzer/pull/769/files#diff-3faa7854e4609c9975566a75ed25c8ea80cc909208f05e038c913ae066a67d2eR573) in test_perf_analyzer_config_generator.py uses `input_data.json`.

Reply:
Thanks for pointing that out. (For testing purposes) It doesn't matter what the file name is in the yaml, just that `input-data` as a PA flag exists.
But, for clarity I've turned input_data into input-data.